modules/core/src/lib/attribute/data-column.ts

```diff
@@ -16,19 +15,16 @@ import log from '../../utils/log';
 import type {TypedArray, NumericArray, TypedArrayConstructor} from '../../types/types';
 
 export type BufferAccessor = {
-  /** A WebGL data type, see [vertexAttribPointer](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/vertexAttribPointer#parameters). */
-  type?: number;
+  /** Vertex data type. */
+  type?: VertexType;
   /** The number of elements per vertex attribute. */
   size?: number;
   /** 1 if instanced. */
-  divisor?: number;
+  divisor?: 1 | 0;
```

Comment:
This [does not have to be](https://developer.mozilla.org/en-US/docs/Web/API/WebGL2RenderingContext/vertexAttribDivisor#divisor) 1 or, I would keep `number`